experimental/examples/chat-mpp/shared/src/commonMain/kotlin/Messages.kt

```diff
@@ -43,14 +43,20 @@ internal inline fun Messages(messages: List<Message>) {
                 ChatMessage(isMyMessage = message.user == myUser, message)
             }
         }
-//        items(messages, key = { it.id }) { message -> //TODO not working in JS
-//            ChatMessage(isMyMessage = message.user == myUser, message)
-//        }
+        if (displayStub) {
+            item {
+                // TODO temporary workaround here
+                // Stub if temporary workaround for better Compose display with SwiftUI interop
+                // For now, Compose view is bigger, then it should be.
+                Box(Modifier.height(250.dp))
+            }
+        }
     }
 }
 
 @Composable
 private inline fun ChatMessage(isMyMessage: Boolean, message: Message) {
+    val focusManager = LocalFocusManager.current
     Box(modifier = Modifier.fillMaxWidth()) {
         Surface(
             modifier = Modifier.padding(4.dp)
```

Comment:
Can you invert the alignment in the modifier below so that it right-aligns `isMyMessage`? That's how almost all messengers layout their messages.

Reply:
Ok, sure

Reply:
Supposed to point at L63, not sure why the comment attached here.

Reply:
Done

<img width="500" alt="image" src="https://user-images.githubusercontent.com/99798741/219739632-0c9b08cc-39a3-4ab9-b570-b2c820b89c04.png">

Reply:
Small update, moved like button to the right:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/99798741/219748131-ad464989-4823-48d9-a6ce-15b691adc3a5.png">